packages/helpers/src/enums.js

```diff
@@ -0,0 +1,51 @@
+export const AlignTypes = {
+  TOP: "top",
+  RIGHT: "right",
+  BOTTOM: "bottom",
+  LEFT: "left",
+};
+
+AlignTypes.ALL = Object.values(AlignTypes);
+
+export const ShirtSizes = {
+  XSMALL: "xsmall",
+  SMALL: "small",
+  MEDIUM: "medium",
+  LARGE: "large",
+  XLARGE: "xlarge",
+};
+
+ShirtSizes.DEFAULT = [ShirtSizes.SMALL, ShirtSizes.MEDIUM, ShirtSizes.LARGE];
```

Comment:
personally I would discourage using this kind of declaration on our propTypes because the consumer will not know what `ShirtSize` means unless goes and look into the code which doesn't seem to be a please developer experience. 

instead, I would put each individual value on the propTypes so the user knowns how to use it.